src/turndown.js

```diff
@@ -6,6 +6,24 @@ import Node from './node'
 var reduce = Array.prototype.reduce
 var leadingNewLinesRegExp = /^\n*/
 var trailingNewLinesRegExp = /\n*$/
+const markdownReplacements = [
+  [/\\/g, '\\\\'],
+  [/\*/g, '\\*'],
+  [/-/g, '\\-'],
+  [/\+/g, '\\+'],
+  [/=/g, '\\='],
+  [/#/g, '\\#'],
+  [/`/g, '\\`'],
+  [/~/g, '\\~'],
```

Comment:
It can also be used for a strikethrough effect:

`~~strikethrough~~` = ~~strikethrough~~

In this case shall we stick with the aggressive escaping for ~?